delivery_postlogistics_dangerous_goods/tests/test_postlogistics_dangerous_goods.py

```diff
@@ -0,0 +1,203 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from os.path import dirname, join
+
+from odoo import exceptions
+from odoo.tests.common import SavepointCase
+
+from vcr import VCR
+
+from ..postlogistics.web_service import PostlogisticsWebServiceDangerousGoods
+
+recorder = VCR(
+    record_mode="once",
+    cassette_library_dir=join(dirname(__file__), "fixtures/cassettes"),
+    path_transformer=VCR.ensure_suffix(".yaml"),
+    filter_headers=["Authorization"],
+    # ignore scheme, host, port
+    match_on=("method", "path", "query"),
+    # allow to read and edit content in cassettes
+    decode_compressed_response=True,
+)
+
+ENDPOINT_URL = "https://wedecint.post.ch/"
+CLIENT_ID = "XXX"
+CLIENT_SECRET = "XXX"
+LICENSE = "XXX"
```

Comment:
Same comment than in https://github.com/OCA/delivery-carrier/pull/358, we really need a common class for these tests.

Reply:
Done